core/src/cats/grant_bareos_privileges.in

```diff
@@ -52,7 +52,7 @@ then
 fi
 
 retval=0
-if [[ "$(uname -s)" =~ _NT ]]; then
+if [ "$(uname -s)" =~ _NT ]; then
```

Comment:
> ShellCheck says "In POSIX sh, =~ regex matching is undefined.", so chances are this will break sooner or later. Maybe we could use `command -v cygpath` to check if `cygpath` is available instead?

would that work also with the new native build and tests ?

Reply:
I think so, yes.

Reply:
After a quick check to decide if we want to go for bash require. Actual Next doesn't require bash under FreeBSD.

Reply:
@pstorz propose eventually to go that way
https://docs.freebsd.org/en/books/porters-handbook/uses/#uses-shebangfix

Reply:
To remove the bashism, we could use the case statement for a substring match, like so:

```
case $(uname -s) in
  *_NT)   i_am_running_in_windows=yes
             ;;
  *)      i_am_running_in_windows=no
            ;;
esac
```

Reply:
So to resume, we decided to keep POSIX scripts for the database, so back to /bin/sh and we use the `case xxx` statement to test the Windows usage.

Reply:
should be fixed correctly now with 2cba3dae6